src/blazar_tempest_plugin/__init__.py

```diff
@@ -0,0 +1,12 @@
+from tempest.lib import auth as _tl_auth
+
+from .auth.oidc_provider import (
+    KeystoneV3OidcAuthProvider,
+    KeystoneV3OidcCredentials,
+)
+
+# Monkey patch the new identity version so tempest will pick it up
+_tl_auth.IDENTITY_VERSION["v3"] = (
+    KeystoneV3OidcCredentials,
+    KeystoneV3OidcAuthProvider,
+)
```

Comment:
It may be possible to specify it as a different identity version, e.g. `v3oidc`, and just update that in the tempest config? but given that this works, I'm not too worried, and could put things like that off to something like a fork of tempest.

Just thinking about how to eventually make the changes a bit more modular.